pages/account.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import styles from "../styles/pages/account.module.scss";
+import AccountUserName from "../components/account/AccountUserName";
+
+export default function Account() {
+	const daysAlive = 9;
+	const [profilePicSrc, setProfilePicSrc] = useState("/lio_img.png");
+
+	const onUpload = (event: React.ChangeEvent<HTMLInputElement>) => {
+		try{
+			if(event.target.files){
+			const file = event.target.files[0];
+			setProfilePicSrc(URL.createObjectURL(file));
+		}else{
+			console.log("no files detected");
+		}}
+		catch {
+			console.log("did not select file");
```

Comment:
a log doesn't help the user to identify the problem, moreover, you don't even display the error that you get...
alert the error instead of the log

Reply:
noted